.ci/setup_env_alinux.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Ant Group
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+cidir=$(dirname "$0")
+source "/etc/os-release" || "source /usr/lib/os-release"
+source "${cidir}/lib.sh"
+
+# Obtain AlibabaCloud Linux version
+# Either /etc/os-release or /usr/lib/os-release is source´ed
+# so that VERSION_ID is already exported.
+alinux_version=$(grep VERSION_ID /etc/os-release | cut -d '"' -f2)
+
+[ "$alinux_version" == 3 ] || die "This script is for alinux 3 only"
+
+# Send error when a package is not available in the repositories
+echo "skip_missing_names_on_install=0" | sudo tee -a /etc/yum.conf
+
+# Ensure EPEL repository is configured
+sudo -E dnf -y install epel-release
+
+# Enable priority to AlibabaCloud Linux Base repo in order to
+# avoid perl updating issues
+if [ -f "/etc/yum.repos.d/AliYun.repo" ]; then
+	repo_file="/etc/yum.repos.d/AliYun.repo"
+fi
+
+[ -n "${repo_file:-}" ] || die "Unable to find the AlibabaCloud Linux base repository file"
+
+if [ "$(tail -1 ${repo_file} | tr -d '\n')" != "priority=1" ]; then
+	echo "priority=1" | sudo tee -a "$repo_file"
+fi
+
+sudo -E dnf -y clean all
+
+echo "Update repositories"
+sudo -E dnf -y --nobest update
+
+echo "FIXME !!!!  Enable PowerTools repository"
+# sudo -E dnf install -y 'dnf-command(config-manager)'
+# sudo dnf config-manager --enable powertools
```

Comment:
Could you either fix this or remove it and optionally add a comment with a GitHub issue URL for the remaining work?

Reply:
deleted